nh_eobs/tests/report_nh_clinical_observation_report/test_report_without_dob.py

```diff
@@ -20,6 +20,9 @@ def new_patient_pool_mock_patient(*args, **kwargs):
 
         self.patient_pool._revert_method('read')
```

Comment:
We need to discuss this.

By removing this line and putting the `revert_method` in the `tearDown` like we should causes the `.nh_eobs/tests/nh_eobs_ward_dashboard/test_ward_dashboard` test to fail on it's `setUpClass` method as evidenced here: https://travis-ci.org/NeovaHealth/openeobs/jobs/189574804

By keeping this it all passes perfectly fine so something is going on with the patient pool